apps/ledger-live-mobile/src/newArch/components/QueuedDrawer/QueuedDrawersContext.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+/**
+ * A function to call when the drawer should be opened or closed.
+ * @param opened Whether the drawer should be opened or closed.
+ */
+export type SetDrawerOpenedCallback = (opened: boolean) => void;
+
+/**
+ * An object that represents a drawer in the queue.
+ * It has methods to remove the drawer from the queue and get its position in the queue.
+ */
+export type DrawerInQueue = {
+  removeDrawerFromQueue: () => void;
+  getPositionInQueue: () => number;
+};
+
+type QueuedDrawersContextType = {
+  /**
+   * Add a drawer to the queue.
+   * @param setDrawerOpenedCallback The callback to call when the drawer should be opened or closed.
```

Comment:
[ASK] Is `setDrawerOpenedCallback` not misleading because this callback is called on a open and a close ?

Reply:
`OnDrawerStateChange` ?

Reply:
Good one thanks 👍